JeMPI_Apps/JeMPI_LibAPI/src/main/java/org/jembi/jempi/libapi/BackEnd.java

```diff
@@ -473,6 +474,21 @@ private Behavior<Event> getSqlDashboardDataHandler(final SQLDashboardDataRequest
       return Behaviors.same();
    }
 
+   private Behavior<Event> getConfigurationHandler(final GetConfigurationRequest request) {
+      String currentPath = System.getProperty("user.dir");
+      String configJsonFilePath = currentPath + "/conf/config-reference.json";
```

Comment:
@harmochiky this needs to be updated and set in the app config as an environment variable